plugins/uefi-capsule/efi/lds/elf_ia64_efi.lds

```diff
@@ -0,0 +1,93 @@
+OUTPUT_FORMAT("elf64-ia64-little")
+OUTPUT_ARCH(ia64)
```

Comment:
I think this linker script can be dropped, since `fwupd` doesn't support `ia64`.

Reply:
Agree.